packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/SpendingCapApproval.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { Accordion, Form } from 'react-bootstrap';
+import { usePassportProvider } from '@/context/PassportProvider';
+import WorkflowButton from '@/components/WorkflowButton';
+import { RequestExampleProps } from '@/types';
+import { Interface } from 'ethers/lib/utils';
+import { useImmutableProvider } from '@/context/ImmutableProvider';
+import { Environment } from '@imtbl/config';
+
+const getErc20DefaultContractAddress = (environment: string) => {
+  switch (environment) {
+    case Environment.SANDBOX:
+      return '0xe9E96d1aad82562b7588F03f49aD34186f996478';
+    case Environment.PRODUCTION:
+      return '0x52a6c53869ce09a731cd772f245b97a4401d3348';
+    default:
+      return '0xba919c45c487c6a7d4e7bc5b42d4ff143a80f041';
```

Comment:
Can we type the arg & replace default with `EnvironmentNames.DEV` please?
```suggestion
const getErc20DefaultContractAddress = (environment: EnvironmentNames) => {
  switch (environment) {
    case EnvironmentNames.SANDBOX:
      return '0xe9E96d1aad82562b7588F03f49aD34186f996478';
    case EnvironmentNames.PRODUCTION:
      return '0x52a6c53869ce09a731cd772f245b97a4401d3348';
    case EnvironmentNames.DEV:
      return '0xba919c45c487c6a7d4e7bc5b42d4ff143a80f041';
```